skops/card/_model_card.py

```diff
@@ -237,21 +237,14 @@ def add_metrics(self, **kwargs: str) -> "Card":
             self._eval_results[metric] = value
         return self
 
-    def save(self, path: str | Path) -> None:
-        """Save the model card.
-
-        This method renders the model card in markdown format and then saves it
-        as the specified file.
-
-        Parameters
-        ----------
-        path: str, or Path
-            filepath to save your card.
+    def _generate_card(self) -> ModelCard:
```

Comment:
This doesn't really generate the card itself because it already exists, but generates the content to be written. That would be better to go for a better naming IMHO.

Reply:
Yes, it doesn't generate the skops `Card`, it generates the modelcards `ModelCard` object. What would be a better name? I think `_generate_content` is a bit vague. I'm also not attached to `generate`, we can also use a plain `make` or something.

Reply:
Since it's not a public method we could improve the docstring instead, I guess. Same with the below one.

Reply:
Happy to improve the docstring. Could you please let me know what I should clarify -- since I made the change, for me it's obvious ;)

> Same with the below one.

Which one do you mean?